frontend/src/modules/tournaments-list/organisms/TournamentTable.js

```diff
@@ -0,0 +1,129 @@
+import { useEffect, useMemo } from 'react';
+import {
+  Table,
+  TableCaption,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@chakra-ui/react';
+import { Button } from '../../../shared/design-system';
+import { useTable, useGlobalFilter } from 'react-table';
+import { tournaments } from '../../schedule-maker/utils/mocks';
+import TournamentListHeading from '../atoms/TournamentListHeading';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const columns = [
+  {
+    Header: 'Tabulka',
```

Comment:
Asi nepotřebujem, aby se to jmenovalo tabulka :D Když tak je to všechno anglicky, tak aspoň table